appeals/api/src/server/endpoints/integrations/integrations.service.js

```diff
@@ -25,27 +28,56 @@ export const produceAppealUpdate = async (
 	/** @type {string} */ updateType
 ) => {
 	const validationResult = await validateFromSchema(schemas.appeal, appeal);
-	if (validationResult === true) {
-		const topic = producers.boCaseData;
-		const res = await eventClient.sendEvents(topic, [appeal], updateType);
-		if (res) {
-			return true;
-		}
-	} else {
-		pino.error('Error producing appeal update', validationResult.errors);
+	if (validationResult !== true && validationResult.errors) {
+		const errorDetails = validationResult.errors?.map(
+			(e) => `${e.instancePath || '/'}: ${e.message}`
+		);
+
+		pino.error(`Error validating appeal: ${errorDetails[0]}`);
+		return false;
 	}
 
+	const topic = producers.boCaseData;
+	const res = await eventClient.sendEvents(topic, [appeal], updateType);
+	if (res) {
+		return true;
+	}
 	return false;
 };
 
+export const broadcastAppealState = async (/** @type {Number} */ id) => {
+	const appeal = await loadAppeal(id);
+	if (appeal) {
+		const appealTopic = mapAppeal(appeal);
+		await produceAppealUpdate(appealTopic, EventType.Update);
+	}
+};
+
 export const produceDocumentUpdate = async (
-	/** @type {any} */ document, // TODO: data and document types schema (PINS data model)
+	/** @type {any[]} */ documents, // TODO: data and document types schema (PINS data model)
 	/** @type {string} */ updateType
 ) => {
-	const topic = producers.boDocument;
-	const res = await eventClient.sendEvents(topic, [document], updateType);
-	if (res) {
-		return true;
+	if (documents.length > 0) {
+		const topic = producers.boDocument;
+		const res = await eventClient.sendEvents(topic, documents, updateType);
+		if (res) {
+			return true;
+		}
+
+		return false;
+	}
+};
+
+export const produceServiceUsersUpdate = async (
+	/** @type {any[]} */ users, // TODO: data and document types schema (PINS data model)
+	/** @type {string} */ updateType
+) => {
+	if (users.length > 0) {
+		const topic = producers.boSvcUsr;
```

Comment:
Very minor, but does this need to be abbreviated? Could it be `boServiceUser` instead?

Reply:
yeah not a problem, it's just an internal label...